Operations_on_polyhedra/include/CGAL/internal/Polyhedron_plane_clipping_3.h

```diff
@@ -131,7 +131,7 @@ Polyhedron clip_to_bbox(const Bbox_3& bbox, const Plane_3& plane)
     if (orientations[i2]==ON_ORIENTED_BOUNDARY) continue;
     if (orientations[i1]!=orientations[i2])
       intersection_points.push_back(
-        get<Point_3>(
+        boost::get<Point_3>(
```

Comment:
Adding the qualification is a bad fix. You no longer use ADL is the call is qualified.

Read this StackOverflow question and best answer:  http://stackoverflow.com/a/2953783/1728537

The best solution is `using boost::get; boost::get<Point_3>(...)`. That way, even if we switch to `std::optional`, the ADL call will remain correct. The `using` is there so that the C++ parser finds at least one `get` function. But ADL will allow to find better matches during the lookup.